### PR TITLE
Testing Pipeline Action

### PR DESCRIPTION
**Pull request by Ieris19**
Developed some Github Workflows in order to deploy the application to Google Cloud's App Engine

### CODE REVIEW

**Review by RyzhasMomentas — APPROVED** [review]
I like

**Comment by Ieris19** [comment]
Decided to remove the workflow that deploys the application to App Engine. 

I believed that the workflow would allow for a fast deployment in a real environment while we moved over to containerizing the application, but in reality, App Engine would require that we set up a whole specific infrastructure so that in the end, app engine can generate a docker container out of it

Decided to cut the middle man, will move on to Docker directly. However, for the time being. The CI pipeline (although we lack tests for it to run) works.

In conclusion, I have removed deployment and all mention of Google Cloud and this PR is in turn only a CI pipeline

**Review by AdrianaLei — APPROVED** [review]
